File: src/codes/Users_Manual/advanced/gpio_out.cpp
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
GPIOOutputs([](const auto& dev,
               const auto& gpio) -> std::optional<GPIOOutputType> {
  if (gpio == GPIOOut::O0)
    return GPIOOutputType::BaseSignal;
  else
    return std::nullopt;
});
//~return 0; }

File: src/codes/Users_Manual/advanced/gpio_out.rs
# use autd3::prelude::*;
# fn main() {
# let _ =
GPIOOutputs::new(|_dev, gpio| {
    if gpio == GPIOOut::O0 {
        Some(GPIOOutputType::BaseSignal)
    } else {
        None
    }
});
# }